lib/cap/vivo/vivo_terms.rb

```diff
@@ -9,7 +9,7 @@ module VivoTerms
       VCARD_givenName  = VCARD_PREFIX + 'givenName'
       VCARD_familyName = VCARD_PREFIX + 'familyName'
 
-      DATA_NAMESPACE = ENV['DATA_NAMESPACE'] || 'https://vivo.stanford.edu'
+      DATA_NAMESPACE = ENV['DATA_NAMESPACE'] || 'https://sul-vivo-dev.stanford.edu'
```

Comment:
This change to the default value should not be necessary.  Just define the ENV value for `DATA_NAMESPACE` instead.